README.md

```diff
@@ -172,6 +172,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [fennel](https://github.com/travonted/tree-sitter-fennel) (maintained by @TravonteD)
 - [x] [fish](https://github.com/ram02z/tree-sitter-fish) (maintained by @ram02z)
 - [ ] [fortran](https://github.com/stadelmanma/tree-sitter-fortran)
+- [ ] [fusion](~/Code/tree-sitter/tree-sitter-fusion)
```

Comment:
please fix this or let the auto-update do its job.

Reply:
uups